tests/data/qtech.json

```diff
@@ -25,8 +25,8 @@
                     "sysObjectID": ".1.3.6.1.4.1.27514.1.1.2.7",
                     "sysDescr": "QSW-3500-24-AC Device, Compiled Feb 22 10:28:35 2011\n SoftWare Version QSW-3500-24-AC_6.0.220.41\n BootRom Version QSW-3500-24-AC_2.0.2\n HardWare Version R01\n Copyright (C) 2010 by QTECH LLC\n All rights reserved",
                     "sysContact": "<private>",
-                    "version": null,
-                    "hardware": null,
+                    "version": "QSW-3500-24-AC_6.0.220.41",
+                    "hardware": "R01",
```

Comment:
Isn't "QSW-3500-24-AC" the hardware?

Reply:
It is reported by one of the switch models as a part of software version. Of course, it is possible to capture just the version number, but I have at hand only two models of switches and have no idea about version string patterns of others. So I would leave it as is for compatibility.

```
SNMPv2-MIB::sysDescr.0 = STRING: QSW-3500-24-AC Device, Compiled Feb 22 10:28:35 2011
 SoftWare Version QSW-3500-24-AC_6.0.220.41
 BootRom Version QSW-3500-24-AC_2.0.2
 HardWare Version R01
 Copyright (C) 2010 by QTECH LLC
 All rights reserved
```
```
SNMPv2-MIB::sysDescr.0 = STRING: QSW-3450-28T-AC Device, Compiled Dec 30 11:22:40 2014
 SoftWare Version 7.0.3.5(R0237.0034)
 BootRom Version 7.1.36
 HardWare Version R01
  Serial No.:00030661509010026177
 Copyright (C) 2014 by QTECH LLC
 All rights reserved
```

Reply:
seems a bit messy   
Either way, `QSW-3500-24-AC` and `QSW-3450-28T-AC` must be the hardware right?  
So you should change that.  

Also, could you add your other device to the tests? You can name the test file like `qtech_qsw3450`

Reply:
ok, I just thought `hardware` is a `revision`, but it is a `platform` actually.